dev/TeachingTip/TeachingTip.idl

```diff
@@ -64,6 +64,14 @@ runtimeclass TeachingTipClosingEventArgs
     Windows.Foundation.Deferral GetDeferral();
 };
 
+
+[MUX_PUBLIC]
+[default_interface]
+[webhosthidden]
+runtimeclass TeachingTipOpenedEventArgs
```

Comment:
I can't think of any information that could be introduced here but maybe other folks know something?